pkg/fanal/analyzer/pkg/apk/apk_test.go

```diff
@@ -453,3 +450,23 @@ func TestParseApkInfo(t *testing.T) {
 		})
 	}
 }
+
+func TestParseLicense(t *testing.T) {
```

Comment:
I think this is a redundant test.
We see this function working in `TestParseApkInfo`.

Reply:
TestLaxSplitLicense was moved to normalize_test in latest commit.